tests/tests_integration/test_ssh_file_transfer.py

```diff
@@ -18,7 +18,7 @@ class TestFileTransfer:
     @pytest.fixture(
         scope="class",
         params=[  # Set running SSH or local filesystem
-            #     False,
+            False,
```

Comment:
I'm confused about why this change was made (and I can't for the life of me figure out where this is used? :rofl: ). 
Does `False` mean SSH or local filesystem or something completely different?
Maybe worth clarifying the comment or adding another comment?

Reply:
(As a side note, and assuming I guess correctly that this change is unrelated to the other changes in the PR which all relate to displaying a message, maybe this should have gone in with a different PR?)

Reply:
Thanks Alessandro, this is very obscure and I've added some docs in the docstrings, let me know if this makes sense:

```
        - Pytest params - The `params` key sets the
        `params` attribute on the pytest `request` fixture.
        This attribute is used to set the `testing_ssh` variable
        to `True` or `False`. In the first run, this is set to
        `False`, meaning local filesystem tests are run. In the
        second run, this is set with a pytest parameter that is
        `True` (i.e. SSH tests are run) but is skipped if `TEST_SSH`
        in `ssh_config` (set in conftest.py` is `False`.
   ```
   
   You are correct this change should not really be in the PR, it is a consequence of me breaking the cardinal rule of 'don't comment out important stuff when testing because you'll forget to uncomment it' in a previous PR. A good reminder to myself never to do this! 😅.